lang/lang_en_de.txt

```diff
@@ -340,7 +340,7 @@
 
 #MSG_FILE_INCOMPLETE c=20 r=2
 "File incomplete. Continue anyway?"
-"Datei unvollstaendig Trotzdem fortfahren?"
+"Datei unvollstaendig. Trotzdem fortfahren?"
```

Comment:
this is another "hack" - there are exactly 20 characters of "Datei unvollstaendig" - that fits on one LCD line exactly, there is no space for the ".", thus it has been skipped. The whole message must fit onto 2 rows.